tests/serverTestMultiMappedPV.conf

```diff
@@ -0,0 +1,28 @@
+eq_conf:
+
+oper_uid:       -1
+oper_gid:       405
+xpert_uid:      1000
+xpert_gid:      1000
+ring_buffer:    10000
+memory_buffer:  500
+
+eq_fct_name:    "PLAIN_VARIABLE_CREATION_TEST._SVR"
+eq_fct_type:    1
+{
+SVR.RPC_NUMBER:         700000004
+SVR.NAME:       "PLAIN_VARIABLE_CREATION_TEST._SVR"
+SVR.BPN:        6000
+}
+eq_fct_name:    "FIRST"
+eq_fct_type:    10
+{
+NAME:   "FIRST"
+SCALAR_1: 12.345
+}
+eq_fct_name:    "SECOND"
+eq_fct_type:    10
+{
+NAME:   "SECOND"
+SCALAR_1: 23.456
```

Comment:
Is this config file intentionally inconsistent? If the server is working, it should keep the SCALAR_1 variables the same.

Reply:
Yes. I added a comment in the test.